events/sunset_invasion.txt

```diff
@@ -70,44 +70,6 @@ character_event = {
 }
 
 # Warcraft
-# Sacrifice Occupied Province Character
-
-character_event = {
-	id = SSI.61
```

Comment:
The event deleted, but still fired in the on-action. Is it intended?

Reply:
Can't confirm

![image](https://user-images.githubusercontent.com/46576860/83526095-99ef7400-a4ee-11ea-9190-3536253a4516.png)

Reply:
Fair, ma bad.